src/useChat.tsx

```diff
@@ -60,6 +61,18 @@ export const useChat = (initValues?: ChatMessage[], initModel?: string) => {
   const [toolCallRegistry] = useState(() => initializeToolCallRegistry());
 
   const [walletStore] = useState(() => new WalletStore());
+  const walletAddress = useWalletStore(walletStore).walletAddress;
+
+  const connectWallet = async () => {
+    await walletStore.connectWallet({
+      chainId: '2222',
+      walletType: WalletTypes.METAMASK,
+    });
+  };
+
+  const disconnectWallet = () => {
+    walletStore.disconnectWallet();
+  };
```

Comment:
Expose these methods to the UI that are currently only implemented in tool calls.